src/models/supplemental_constructors.jl

```diff
@@ -113,6 +113,23 @@ function FACTSControlDevice(
     )
 end
 
+"""Allows construction of Impedance Correction Table data with control modes and winding association."""
+function ImpedanceCorrectionData(
+    table_number,
+    impedance_correction_function_data,
+    transformer_winding::String,
+    transformer_control_mode::String,
+    internal = InfrastructureSystemsInternal(),
+)
+    return ImpedanceCorrectionData(
+        table_number,
+        impedance_correction_function_data,
+        get_enum_value(WindingCategory, transformer_winding),
```

Comment:
It looks like this only gets used with a non-constant once? Feel free to disagree but it seems better to convert the winding number to a `WindingCategory` in that one place rather than to define a new constructor for it. I think the motivating principle here is we should convert from not inherently meaningful numbers to very inherently meaningful enum values as soon as possible.